diff --git a/libgomp/configure.tgt b/libgomp/configure.tgt
index b88bf72fe3d..d37f7b2bd65 100644
--- a/libgomp/configure.tgt
+++ b/libgomp/configure.tgt
@@ -68,7 +68,7 @@ if test x$enable_linux_futex = xyes; then
 
     # Note that bare i386 is not included here.  We need cmpxchg.
     i[456]86-*-linux*)
-	config_path="linux/x86 linux posix"
+	config_path="linux posix"
 	case " ${CC} ${CFLAGS} " in
 	  *" -m64 "*|*" -mx32 "*)
 	    ;;
@@ -83,7 +83,7 @@ if test x$enable_linux_futex = xyes; then
     # can't rely on the --with-arch configure option, since that
     # applies to the 64-bit side.
     x86_64-*-linux*)
-	config_path="linux/x86 linux posix"
+	config_path="linux posix"
 	case " ${CC} ${CFLAGS} " in
 	  *" -m32 "*)
 	    XCFLAGS="${XCFLAGS} -march=i486 -mtune=generic"
